content/projects/publii.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Publii
+repo: GetPublii/Publii
+homepage: https://getpublii.com
+language:
+  - JavaScript
+license:
+  - GPL 3.0
+templates:
+  - Handlebars
+description: desktop-based CMS for Windows and Mac that makes creating static websites fast and hassle-free
+twitter: getpublii
+startertemplaterepo: GetPublii/theme-Starter
```

Comment:
"starter template" means a repo that can be deployed as is with a CLI command. You'll want to provide a repo that works that way or remove this line.